src/editors/ADASEditor.jsx

```diff
@@ -5,6 +5,38 @@ import EditableControl from '../form-components/EditableControl';
 import { handleFormUpdate } from '../utils';
 
 export default class ADASEditor extends React.Component {
+
+    formatLineStd(id, checked, text) {
+        return (
+            <div className="form-check">
+                <label className="form-check-label">
+                    <input
+                        id={id}
+                        className="form-check-input"
+                        type="checkbox"
+                        onChange={handleFormUpdate.bind(this)}
+                        checked={checked}/>
+                    {text}
+                </label>
+            </div>
+        );
+    }
+
+    formatControlStd(className, id, name, value) {
+        return (
+            <div className={className}>
+                <EditableControl
+                    id={id}
+                    name={name}
+                    value={value}
+                    valueEditable={true}
+                    isInstructor={this.props.isInstructor}
+                    updateGraph={this.props.updateGraph}
+                />
+            </div>
+        );
+    }
```

Comment:
This abstraction lends itself less to a custom component - since we're already using a custom component here. We're really just containing it inside a container div. This function looks good I would just make some minor adjustments to make it more semantic.

```
    makeContainedEditableControl(containerClass, id, name, value) {
        return (
            <div className={containerClass}>
                <EditableControl
                    id={id}
                    name={name}
                    value={value}
                    valueEditable={true}
                    isInstructor={this.props.isInstructor}
                    updateGraph={this.props.updateGraph}
                />
            </div>
        );
    }
```